adapters/adapters-base/src/main/java/org/apache/cassandra/sidecar/adapters/base/db/ConnectedClientStatsDatabaseAccessor.java

```diff
@@ -43,7 +45,7 @@ public ConnectedClientStatsDatabaseAccessor(CQLSessionProvider sessionProvider,
      */
     public ConnectedClientStatsSummary summary()
     {
-        tableSchema.prepareStatements(session());
+        Preconditions.checkState(tableSchema.isInitialized(), tableSchema.getClass().getSimpleName() + " is not initialized yet");
```

Comment:
NIT : avoid string concat for the happy path:
```suggestion
        Preconditions.checkState(tableSchema.isInitialized(), () -> tableSchema.getClass().getSimpleName() + " is not initialized yet");
```